source/config/configSpec.py

```diff
@@ -56,6 +56,7 @@
 # Audio settings
 [audio]
 	audioDuckingMode = integer(default=0)
+	wasapi = boolean(default=false)

```

Comment:
The "audio" section is saved in profiles. However it seems to me that this options is only relevant for the base profile since it takes effect after restart.

Would it be worth to put this option in another or a new section (e.g. global settings / startup settings)?

Note that there is probably already some confusion on this point regarding some advanced settings; I am specifically thinking to "Registration for UI Automation events and property changes" option; but there may be others in adv settings.

Reply:
I agree this could be a point of confusion. That said, it seems overkill to create an entirely new config section for a setting which will eventually be removed. (I don't think it makes sense to keep both audio backends long-term.) It also doesn't fit so well in any of the existing global sections. Nevertheless, I'm of course happy to change this if the reviewer has a specific recommendation.